README.md

```diff
@@ -16,46 +16,43 @@ This is an example Solo.io Gloo Platform Dev Portal frontend app, built with [Vi
    mkdir portal-test && cd portal-test && npx tmplr solo-io/dev-portal-starter#main
    ```
 
-3. Run the following to output a docker image (replace username):
+3. Create a `.env.local` file in the `projects/ui` folder. Replace environment variable values to match your Gloo Platform Portal and oauth provider's installation. This file is ignored by git.
 
    ```shell
-   IMAGE_NAME=username/portal-frontend make build-ui-image
+   VITE_PORTAL_SERVER_URL="/v1"
+   VITE_CLIENT_ID="your-client-id"
+   VITE_CLIENT_SECRET="your-client-secret"
+   VITE_TOKEN_ENDPOINT="your-token-endpoint"
    ```
 
-_\*\*\* Note: If building the docker image without `make build-ui-image`, make sure to run `make build-ui` first to get the UI changes.\*\*\*_
-
-4. Push your docker image:
+4. Run the following to output a docker image, replacing the image name.
 
    ```shell
-   docker push username/portal-frontend
+   IMAGE_NAME="your-image-name" \
+   make build-ui-image
    ```
 
-5. Follow [the instructions in the engineering-demos repo](https://github.com/solo-io/engineering-demos/blob/ad5f6e217a50c8fcc9d1aa6e442a2c9bbef47eb2/gloo-mesh/portal/multicluster/README.md) to set up dev portal resources. Use the same image name that you used to build the image for the portal-frontend deployment's `spec.template.containers.image` field.
+_\*\*\* Note: If building the docker image without `make build-ui-image`, make sure to run `make build-ui` with your environment variables set first to get the UI changes.\*\*\*_
 
-## UI Iteration
-
-The following steps can be used to iterate on the UI after the initial setup. These steps must be repeated after each UI change to see them in the build, and can be added as a `.bashrc` or `.zshrc` [alias](https://linuxize.com/post/how-to-create-bash-aliases/). To see UI changes faster, see the next section on "UI Iteration with Storybook and Mock Data".
-
-1. Build the docker image (replace "username").
+4. Push your docker image:
 
    ```shell
-   IMAGE_NAME=username/portal-frontend make build-ui-image
+   docker push "your-image-name"
    ```
 
-2. Push your docker image (replace "username").
+5. Follow [the instructions in the engineering-demos repo](https://github.com/solo-io/engineering-demos/blob/ad5f6e217a50c8fcc9d1aa6e442a2c9bbef47eb2/gloo-mesh/portal/multicluster/README.md) to set up dev portal resources. Use the same image name that you used to build the image for the portal-frontend deployment's `spec.template.containers.image` field.
```

Comment:
is this a public repo?? I thought our engineering-demos were only internal

Reply:
It is, but the public docs links aren't ready yet - I'll need to do another pass to update them when they are. Just added this issue to address it then: https://github.com/solo-io/gloo-mesh-enterprise/issues/9708